### PR TITLE
restored

### PR DESCRIPTION
**Pull request by nafisa-valieva**
Here is my first attempt to create pull request. I have created 14 files for persons attested in CCR 7b of 
DabraLibanosHamGG1.

### CODE REVIEW

File: new/PRS12838Yetbarak.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ይትባረክ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Yǝtbārak</persName>
+                    <persName xml:lang="gez"><roleName type="title">መልሐዘ፡</roleName></persName>
+                      <persName xml:lang="gez"><roleName type="title">malḥaza</roleName></persName>
```
Comment (PietroLiuzzo) [inline new/PRS12838Yetbarak.xml]: please add @type='normalized'

File: new/PRS12837AbYekun.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">አብ፡ይኩን፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAb Yǝkun</persName>
+                    <persName xml:lang="gez"><roleName type="title">ቀይሰ፡ገበዝ፡ጽዮን፡</roleName></persName>
+                      <persName xml:lang="gez"><roleName type="title">qaysa gabaz Ṣǝyon</roleName></persName>
```
Comment (PietroLiuzzo) [inline new/PRS12837AbYekun.xml]: please add type=normalized

File: new/PRS12839ZaAdonay.xml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዘአዶናየ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Za-’Adonāy</persName>
+                    <persName xml:lang="gez"><roleName type="title">ቀስ፡ሐጺን፡</roleName></persName>
+                      <persName xml:lang="gez"><roleName type="title">qas ḥaṣin</roleName></persName>
```
Comment (PietroLiuzzo) [inline new/PRS12839ZaAdonay.xml]: please add type = normalized

File: new/PRS12836Sawen.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                  <persName xml:lang="gez">  <roleName type="title">ሊቀ፡ዲያቆናት፡</roleName></persName>
+                    <persName xml:lang="gez" type="normalized"><roleName type="title">liqa diyaqonāt</roleName></persName>
+                    <faith type="Christianity"/>
+                    <occupation type="ecclesiastic"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS12836Sawen" passive="DabraLibanosHamGG1"/>
```
Comment (PietroLiuzzo) [inline new/PRS12836Sawen.xml]: I am not sure about the use of lawd:hasAttestation here, could you point me to the page of the Guidelines where you have taken this from? thanks

File: new/PRS12839ZaAdonay.xml (hunk 3 above)
Comment (PietroLiuzzo) [reply new/PRS12839ZaAdonay.xml]: please check in all files

File: new/PRS12838Yetbarak.xml (hunk 1 above)
Comment (nafisa-valieva) [reply new/PRS12838Yetbarak.xml]: It is from online-generated schema

File: new/PRS12836Sawen.xml (hunk 4 above)
Comment (nafisa-valieva) [reply new/PRS12836Sawen.xml]: lawd:hasAttestation  is from online-generated schema

File: new/PRS12837AbYekun.xml (hunk 2 above)
Comment (nafisa-valieva) [reply new/PRS12837AbYekun.xml]: Pietro, here I was Following this schema, where you want me to add type=normalized?
 `<persName xml:lang="gez" xml:id="n1" type="birth">ሚናስ፡</persName>
    <persName xml:lang="gez" corresp="#n1" type="normalized">Minās</persName>
    <persName><roleName type="title">ʾaṣe</roleName></persName>`

**Comment by PietroLiuzzo** [comment]
ok.

**Comment by PietroLiuzzo** [comment]
Dear Nafisa.
now your PR is merged into the master. Please do not feel in a rush to make PR.
update your master branch locally, then create a new branch in this repository from there if you need to add or modify data.

**Pull request merged**